test/prob/generate_tests.cpp

```diff
@@ -25,44 +25,37 @@ vector<string> lookup_argument(const string& argument, const int& ind) {
     args.push_back("int");
     args.push_back("std::vector<int>");
     args.push_back("Eigen::Matrix<int, Eigen::Dynamic, 1>");
-    args.push_back("Eigen::Matrix<int, 1, Eigen::Dynamic>");
   } else if (iequals(argument, "double")) {
     args.push_back("double");
     args.push_back("var");
   } else if (iequals(argument, "doubles")) {
     args.push_back("double");
     args.push_back("std::vector<double>");
     args.push_back("Eigen::Matrix<double, Eigen::Dynamic, 1>");
-    args.push_back("Eigen::Matrix<double, 1, Eigen::Dynamic>");
```

Comment:
Let's find a way to have a choice of running them or not. Doing this eliminates the option, which is not what we want to do.

Reply:
I was hoping to add a more user-friendly way back in later with the complete rewrite; for now I can run the old version with `git revert d12cda2; ./runTests -j8 test/prob` or whatever. I don't think anyone besides me is going to want to run the old version between now and then, and you always have source control to go back to the status quo if you want.

Reply:
Perfect. Can you set up Jenkins to run that on develop? If we do that, I think we'll be safe for now. 

Can you open up the issue and we can start putting up requirements? I want to make sure we cover enough functionality.